parcelable/src/commonMain/kotlin/com/arkivanov/essenty/parcelable/IgnoredOnParcel.kt

```diff
@@ -0,0 +1,9 @@
+package com.arkivanov.essenty.parcelable
+
+/**
+ * A multiplatform alias for `kotlinx.parcelize.IgnoredOnParcel` from the `kotlin-parcelize` Gradle plugin.
+ * Make sure you have the `kotlin-parcelize` Gradle plugin applied.
+ */
+@Target(AnnotationTarget.PROPERTY)
+@Retention(AnnotationRetention.SOURCE)
+expect annotation class IgnoredOnParcel
```

Comment:
Let's annotate it also with the following, so there is no need to add the empty annotation in the `nonAndroidMain` source set. See `Parceler` or `Parcelize` for an example.

```kotlin
@OptIn(ExperimentalMultiplatform::class)
@OptionalExpectation
```